src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -1338,14 +1338,16 @@ TEST_CASE("contract storage", "[tx][soroban]")
         auto autobump = enableAutobump();
         REQUIRE(autobump > 1);
 
-        // Check that manual bump doesn't go over max
+        // Check that attempting to bump past max ledger results in error
         put("key", 0, ContractDataDurability::PERSISTENT);
-        bump("key", ContractDataDurability::PERSISTENT, UINT32_MAX);
+        bumpWithFootprint(
+            "key", UINT32_MAX, contractKeys,
+            {contractDataKey(contractID, makeSymbolSCVal("key"),
+                             ContractDataDurability::PERSISTENT, DATA_ENTRY)},
+            false, ContractDataDurability::PERSISTENT);
 
         auto maxExpiration =
             ledgerSeq + stateExpirationSettings.maxEntryExpiration - 1;
-        checkContractDataExpirationLedger(
```

Comment:
Instead of removing this, I would do a bump to the max and check that it worked.